src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+
+.body {
+  margin: 0;
+}
```

Comment:
You are using the `body` tag for styling. According to the checklist, you should avoid using tag names for styling (except `html` and `body`). But in your case, you have used a class `.body` instead of the `body` tag. It's a bit confusing. Please, rename the class to something more meaningful or use the `body` tag if you meant to style the whole body of the document.